### ISSUE TITLE
Change Completion Statuses default sort order to a more logical sorting

### ISSUE THREAD

**Issue by darklinkpower** [JosefNemec/Playnite#1951]
**Describe the solution you'd like**

Currently the sort order doesn't seem to follow any logical sorting; it seems somewhat random and doesn't follow rules.

![imagen](https://user-images.githubusercontent.com/1389286/89667195-570c9c80-d8a1-11ea-8a0a-eb1b9bf9e130.png)

I propose to changing a more logical order that groups and follows the usual game playing order: `Not Played` -> `Playing` -> `Finished`

- Not Played group
  - Not Played
  - Plan to Play
- Playing group
  - Played
  - On Hold
  - Playing
- Finished group
  - Abandoned
  - Beaten
  - Completed

![imagen](https://user-images.githubusercontent.com/1389286/89667325-920ed000-d8a1-11ea-8c88-18b388014457.png)

**Comment by shakeyourbunny** [comment]
I'd add at least one another new completion status, namely one for these all endless games like MMOs / Diablo where the game is perpetually updated and new content is patched in. These games are never really finished until their (online) servers are down.

In addition, there are also some games which are meant to be played for a short burst of time, high score games like Tetris, Pac-Man, these PopCap games and their like. They are never "finished". 

Party games meant for playing with other people is also not covered, like ... how do you "complete" Mario Party, a brawler like Super Smash Brothers? They would stay eternally in "played".

In this vein, I'd suggest some sort of new statuses like "recurring play" or something similar.

**Comment by darklinkpower** [comment]
You know, I thought before that "Playing" would suffice fo those cases, but after using the completion status for a while to filter, I realized just how needed is a completion status for what you mention, specially for endless multiplayer games in my case. I was thinking what would be a good name but couldn't come up with anything, but I like the name "Recurrent playing" that you mention. I would place it between "Playing" and "Abandoned".

**Comment by JosefNemec** [comment]
I'm closing this because it's going to be obsolete after #1388 is implemented.
